### DISCUSSION TITLE
Data Tables- Replace a feature-column with a meta-column

### DISCUSSION THREAD

**Discussion by ozi42** [biolab/orange3#5646] [Q&A]
Hello, 

i have a data table. The Data Table has six features. 
Is it possible to declare the first two features-column as meta-colums?

-------------------------------------
CODE-Example:

data=copy.deepcopy(in_data)
print(data)

<< [[1.0, 2, 5, 3, 3, 9],
<< [1.0, 5, 10, 2, 4, 9],
<< [3.0, 5, 2, 5, 1, 1],
<< [0.9, 4, 5, 8, 8, 5]]

print(data.domain)

<<[ Jan, Feb, März, Apr, Mai, Juni]

-------------------------------------
 

Related to this example : The Features Jan and Feb should be Metas.

**Comment by ajdapretnar** [comment]
Are you referring to the code example?
Also, this is more appropriate as a discussion, not an issue, so moving it there.

**Comment by ajdapretnar** [comment]
Here's how to do it in code:

```
>>> import Orange
>>> 
>>> iris = Orange.data.Table("iris")
>>> new_domain = Orange.data.Domain(iris.domain.variables[:2], metas=iris.domain.variables[2:])
>>> new_domain
[sepal length, sepal width] {petal length, petal width, iris}
>>> new_table = iris.transform(new_domain)
>>> new_table
[[5.1, 3.5] {1.4, 0.2, Iris-setosa},
 [4.9, 3.0] {1.4, 0.2, Iris-setosa},
 [4.7, 3.2] {1.3, 0.2, Iris-setosa},
 [4.6, 3.1] {1.5, 0.2, Iris-setosa},
 [5.0, 3.6] {1.4, 0.2, Iris-setosa},
 ...
]
```
